commands/host.vegman

```diff
@@ -304,8 +310,10 @@ function cmd_virtualmedia_umount {
   busctl call xyz.openbmc_project.VirtualMedia \
     "/xyz/openbmc_project/VirtualMedia/${dbus_vm_type}/${media}" \
     "xyz.openbmc_project.VirtualMedia.${dbus_vm_type}" Unmount > /dev/null && \
-  echo "Virtual Media (${media}) unmounted" || \
-  echo "Failed to unmount Virtual Media (${media})"
+  echo "Virtual Media (${media}) unmounted" && \
+  log_audit "Virtual Media (${media}) unmounted" || \
+  echo "Failed to unmount Virtual Media (${media})" && \
+  log_audit "Failed to unmount Virtual Media (${media})"
```

Comment:
This again needs parentheses or it won't do what you expect.
`cmd & (success; actions) || (failure; actions)`